### PR TITLE
feat: Add full validation rules and vehicle types for TROS/TROW

### PR DESCRIPTION
**Pull request by john-fletcher-aot**
# Description

Add a master configuration to represent the complete, current policy config for all permit types. Add all vehicle types and all validation rules for TROS and TROW to this master config.

Fixes # ORV2-2219, ORV2-2220

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

- [X] Added unit tests for validation of a valid TROS and a valid TROW against the master config

## Checklist

- [X] I have read the [CONTRIBUTING](CONTRIBUTING.md) doc
- [X] I have performed a self-review of my own code
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] I have made corresponding changes to the documentation
- [X] My changes generate no new warnings
- [X] I have added tests that prove my fix is effective or that my feature works
- [X] New and existing unit tests pass locally with my changes
- [X] Any dependent changes have already been accepted and merged

---

Thanks for the PR!

Deployments, as required, will be available below:
- [Frontend](https://onroutebc-1449-frontend.apps.silver.devops.gov.bc.ca)
- [Vehicles](https://onroutebc-1449-vehicles.apps.silver.devops.gov.bc.ca/api)
- [Dops](https://onroutebc-1449-dops.apps.silver.devops.gov.bc.ca/api)
- [Scheduler](https://onroutebc-1449-scheduler.apps.silver.devops.gov.bc.ca/api)

Please create PRs in draft mode.  Mark as ready to enable:
- [Analysis Workflow](https://github.com/bcgov/onroutebc/actions/workflows/analysis.yml)

After merge, new images are promoted to:
- [Merge Workflow](https://github.com/bcgov/onroutebc/actions/workflows/merge.yml)

### CODE REVIEW

**Review by praju-aot — APPROVED** [review]

File: policy/test/permit-app/valid-tros-30day.ts [hunk 1]
```diff
@@ -31,8 +31,8 @@ export const validTros30Day: PermitApplication = {
       phone1Extension: null,
       phone2: '(250) 555-4321',
       phone2Extension: null,
-      email: 'bryan.lane@gov.bc.ca',
-      additionalEmail: 'michael.melo@gov.bc.ca',
+      email: 'chief.bakergov.bc.ca',
```
Comment (praju-aot) [inline policy/test/permit-app/valid-tros-30day.ts]: Incorrect email syntax

**Comment by zgong-gov** [comment]
How should I make use of this policy engine and its validations on the frontend? Does it generate a config json or something?

File: policy/test/permit-app/valid-tros-30day.ts (hunk 1 above)
Comment (john-fletcher-aot) [reply policy/test/permit-app/valid-tros-30day.ts]: Thanks Praveen, fixed the email address

**Comment by john-fletcher-aot** [comment]
> How should I make use of this policy engine and its validations on the frontend? Does it generate a config json or something?

I will be writing up usage docs in the README shortly, though we haven't yet determined how we will share the package between fe and be. My ideal scenario is that you would npm install orbc-policy-engine, then you'll have the library available to use. Construct a new PolicyEngine by passing in a policy configuration, then using that PolicyEngine instance call a validate() method passing in the permit application JSON. Return type of the validate method (async) will be a ValidationResults object with arrays of individual results which could represent field data violations, invalid duration, invalid vehicle type for the permit type, etc.

But, this will be more clear in the README once I get it in there!

**Pull request merged**